datasets/timit_asr/timit_asr.py

```diff
@@ -0,0 +1,171 @@
+
+import os
+
+import pandas as pd
+
+import datasets
+
+
+_CITATION = """\
+@inproceedings{
+  title={TIMIT Acoustic-Phonetic Continuous Speech Corpus},
+  author={Garofolo, John S., et al},
+  ldc_catalog_no={LDC93S1},
+  DOI={https://doi.org/10.35111/17gk-bn40},
+  journal={Linguistic Data Consortium, Philadelphia},
+  year={1983}
+}
+"""
+
+_DESCRIPTION = """\
+The TIMIT corpus of reading speech has been developed to provide speech data for acoustic-phonetic research studies
+and for the evaluation of automatic speech recognition systems.
+
+TIMIT contains high quality recordings of 630 individuals/speakers with 8 different American English dialects,
+with each individual reading upto 10 phonetically rich sentences.
+
+More info on TIMIT dataset can be understood from the "README" which can be found here:
+https://catalog.ldc.upenn.edu/docs/LDC93S1/readme.txt
+"""
+
+_URL = "https://data.deepai.org/timit.zip"
+
+
+class TimitASRConfig(datasets.BuilderConfig):
+    """BuilderConfig for TimitASR."""
+
+    def __init__(self, **kwargs):
+        """
+        Args:
+          data_dir: `string`, the path to the folder containing the files in the
+            downloaded .tar
+          citation: `string`, citation for the data set
+          url: `string`, url for information about the data set
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(TimitASRConfig, self).__init__(version=datasets.Version("2.0.1", ""), **kwargs)
+
+
+class TimitASRConfig(datasets.GeneratorBasedBuilder):
+    """TimitASR dataset."""
+
+    BUILDER_CONFIGS = [
+        TimitASRConfig(name="clean", description="'Clean' speech."),
+    ]
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "file": datasets.Value("string"),
+                    "text": datasets.Value("string"),
+                    "phonetic_detail": datasets.Sequence(
+                        {
+                            "start": datasets.Value("int64"),
+                            "stop": datasets.Value("int64"),
+                            "utterance": datasets.Value("string"),
+                        }
+                    ),
+                    "word_detail": datasets.Sequence(
+                        {
+                            "start": datasets.Value("int64"),
+                            "stop": datasets.Value("int64"),
+                            "utterance": datasets.Value("string"),
+                        }
+                    ),
+                    "dialect_region": datasets.Value("string"),
+                    "sentence_type": datasets.Value("string"),
+                    "speaker_id": datasets.Value("string"),
+                    "id": datasets.Value("string"),
+                }
+            ),
+            supervised_keys=("file", "text"),
+            homepage=_URL,
```

Comment:
_URL is the url to download the file, not the homepage.
Is there a valid homepage we can use here instead ?